### DISCUSSION TITLE
Any help wanted?

### DISCUSSION THREAD

**Discussion by Fiwx** [PGScatalog/pgsc_calc#251] [Ideas]
Hi,
I appreciate the work the team puts in for this open source project. I have not contributed to open source projects before so please excuse my ignorance. Are there any tasks or updates you'd like help working on?

**Comment by smlmbrt** [comment]
Hi @Fiwx, we haven't made a code of conduct or contribution guidelines yet but it's very much on our radar. We track most of our development goals and issues on here, so if you see something that you could solve and want to work on do reply and we can double-check that no one on our end is already doing it (but we do try to assign).
